MAUI/Polar-Charts/PolarLine.md

```diff
@@ -0,0 +1,242 @@
+---
+layout: post
+title: Polar Line Chart in .NET MAUI Chart control | Syncfusion
+description: Learn here all about the Polar line chart type and its features in the Syncfusion .NET MAUI Chart (SfPolarChart) control.
+platform: maui
+control: SfPolarChart
+documentation: ug
+---
+
+# Polar Line Chart in .NET MAUI Chart
+
+## Polar Line Chart
+
+To display an line series in a polar chart, instantiate the [PolarLineSeries]() and include it in the [Series]() collection property of [SfPolarChart]().
```

Comment:
To display a line series in a polar chart, instantiate the [PolarLineSeries]() and include it in the [Series]() collection property of [SfPolarChart]().